src/mongoc/mongoc-cursor-private.h

```diff
@@ -46,7 +46,13 @@ struct _mongoc_cursor_interface_t
    void             (*get_host) (mongoc_cursor_t        *cursor,
                                  mongoc_host_list_t     *host);
 };
+#ifdef MIN
+#undef MIN
+#endif
 
+#ifdef MAX
+#undef MAX
+#endif
```

Comment:
Hello @bjori 
I can update this PR.

I will try to rename, but it comes from HP system header, and probably no luck with it.